src/lambda/commands/deleteCloudFormation.ts

```diff
@@ -18,10 +18,17 @@ export async function deleteCloudFormation(
     node?: CloudFormationStackNode
 ) {
     if (!node) {
+        vscode.window.showErrorMessage(
```

Comment:
In what situation can this occur? Is there any guidance that we can provide in the error message for how to resolve it?

Reply:
More of a technicality - you can assign a key map to any command, which would trigger this with an undefined node.

This is the only way I'm aware that this code could run that isn't tied to the context menu